File: Dataset/Leetcode/train/9/93.cpp
class Solution {
public:
    bool XXX(int x) {
        if(x<0) return false;
        string ss=to_string(x);
        for(int i=0;i<ss.length()/2;i++)
        {
            if(ss[i]!=ss[ss.length()-i-1])
             return false;
        }
        return true;

    }
};

File: Dataset/Leetcode/train/101/93.java
 class Solution {
    public boolean XXX(TreeNode root) {
        return check(root,root);
    }
    public boolean check(TreeNode q,TreeNode p) {
        if(q == null && p == null) {
            return true;
        }
        if(q == null || p == null) {
            return false;
        }
        if(q.val == p.val && check(q.left,p.right) && check(q.right,p.left)) {
            return true;
        }
        return false;
    }
}

